README.md

```diff
@@ -1 +1,55 @@
-# Leaderboad
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Leaderboard:
+
+## Live Demo
+
+[Link to Live demo](https://ridwanediallo.github.io/To-do-List-App/dist/)
+
+# Getting Started
+
+To get a local copy up and running follow these simple steps.
+
+- You can clone this repo by typing on your terminal.<pre><code>git clone https://github.com/ridwanediallo/Leaderboad.git</code></pre>
+- Type <pre><code>cd To-do-List-App</code></pre> to access the project on the terminal
+- Run: <pre><code>npm install</code></pre> to have all the project dependencies in your local environment
+
+- Run: <pre><code>npm start</code></pre> to view the output in your browser
+
+- Run: <pre><code>npm run watch</code></pre> to watch for the changes
+
+## Built With
+
+- Major languages
+- HTML and CSS
+- Mostly JavaScript
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+Browser. Any modern browser should do the job.
+
+### Setup
+
+Just download or clone the repo and run the index.html file.
+
+## Author
+
+👤 **Ridwan**
+
+- GitHub: [@githubhandle](https://github.com/ridwanediallo)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Report an issue
+
+-Go here 👉 [@Link](https://github.com/ridwanediallo/Leaderboad/issues)
```

Comment:
- [ ] From the requirements of this project, we are needed to also include the API(s) it consumes.

Reply:
> * [ ]  From the requirements of this project, we are needed to also include the API(s) it consumes.

Hello, thank you for the review. However, this request is not in the requirement.
You can check it out here: https://github.com/microverseinc/curriculum-javascript/blob/main/leaderboard/m3_final_touches.md